pkgs/tools/misc/mcfly/default.nix

```
@@ -21,7 +21,7 @@ rustPlatform.buildRustPackage rec {
 
   meta = with lib; {
     homepage = "https://github.com/cantino/mcfly";
-    description = "An upgraded ctrl-r for Bash whose history results make sense for what you're working on right now";
+    description = "An upgraded ctrl-r where history results make sense for what you're working on right now.";
```

```suggestion
    description = "An upgraded ctrl-r where history results make sense for what you're working on right now";
```

No period per https://nixos.org/manual/nixpkgs/stable/#var-meta-description

---

Done.